### ISSUE TITLE
OpenAL Soft 1.19.0 EFX  error

### ISSUE THREAD

**Issue by barnes14** [kcat/openal-soft#224]
After the upgrade, I can not initiate EFX

```
efx.rvbAuxSlot = 0;
alGenAuxiliaryEffectSlots (1, &efx.rvbAuxSlot); --->>alGetError return OUT_OF_MEMORY
```

**Comment by kcat** [comment]
It's working here. Do the alreverb and almultireverb examples work for you? How many effect slots are you trying to create? Do you have any config settings that may limit the number of effect slots? If you log warnings (set the `ALSOFT_LOGLEVEL` environment variable to `2` or `3`, and get the output for stderr), what does it say about the error?

**Comment by barnes14** [comment]
im use one efx slot (1.18.2 worked fine for me) and change only efx presets

https://sourceforge.net/p/quake2xp/code/HEAD/tree/trunk/client/snd_efx.c

**Comment by kcat** [comment]
Checking to see if the reverb examples (or even openal-info's effect info) works would help, as would seeing the output from a warning or trace log of the game.

**Comment by kcat** [comment]
Should be fixed now, with commit a6734c7a91b1e2f2ef19ba163ceb2cb14571a9dd. It wasn't accounting for empty (deleted) effect slot list elements when determining how many more can be allocated (though it would reuse the empty elements when allocating, so it's only a problem if you try to allocate more than the limit minus the number of concurrent effect slots you've ever had allocated).

**Comment by barnes14** [comment]
All work fine now. Thx
